arbitrum-docs/how-arbitrum-works/03-sequencer.mdx

```diff
@@ -259,3 +260,28 @@ Developers and users should consider the appropriate level of finality based on
 ### Decentralized Fair Sequencing
 
 Arbitrum’s long-term vision includes transitioning from a centralized Sequencer to a decentralized, fair sequencing model. In this framework, a committee of servers (or validators) collectively determines transaction ordering, ensuring fairness, reducing the influence of any single party, and making it more resistant to manipulation. By requiring a supermajority consensus, this approach distributes sequencing power among multiple honest participants, mitigates the risks of front-running or censorship, and aligns with the broader blockchain principles of enhanced security, transparency, and decentralization.
+
+## Censorship Timeout
```

Comment:
I think it makes sense to move this section on top of ### Decentralized Fair Sequencing section.